pom.xml

```diff
@@ -3,7 +3,7 @@
 	<groupId>io.magentys</groupId>
 	<artifactId>donut-maven-plugin</artifactId>
 	<packaging>maven-plugin</packaging>
-	<version>0.0.4-SNAPSHOT</version>
+	<version>0.0.5-SNAPSHOT</version>
```

Comment:
No need to bump the version manually. This is done automatically by the maven release plugin as part of the release.